impeller/playground/playground_impl.cc

```diff
@@ -19,6 +20,50 @@
 #include "impeller/playground/backend/vulkan/playground_impl_vk.h"
 #endif  // IMPELLER_ENABLE_VULKAN
 
+namespace {
+static const std::vector<std::string> kVulkanDenyValidationTests = {
```

Comment:
Let's share this instead of duplicating it twice.

Reply:
Done